src/languageFeatures.ts

```diff
@@ -218,64 +217,66 @@ function toTextEdit(textEdit: ls.TextEdit): editor.ISingleEditOperation {
   };
 }
 
-export class CompletionAdapter implements languages.CompletionItemProvider {
-  triggetCharacters = [' ', ':'];
```

Comment:
This typo had gone unnoticed in the class implementation, but was caught because it doesn’t match the interface returned by the factory function.